input/ivr-benchmark-twilio-autopilot.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Place all the IVR input files under a sub-directory "input/ivr", and all the nlp ones under "input/nlp"